openapi-mock/README.md

```diff
@@ -5,131 +5,30 @@
 # OpenAPI Mock
 [![npm version](https://badge.fury.io/js/varkes-openapi-mock.svg)](https://badge.fury.io/js/varkes-openapi-mock)
 
-Mocks res calls given an OpenAPI specification 
+The OpenAPI mock mocks the application APIs based on the OpenAPI specification.
 
-After startup, you have access to:
-- rest API - http://localhost:10000/myApiBasePath/metadata
-- console - http://localhost:10000/myApiBasePath/console
+## Features
 
-## Starting the application
+The OpenAPI mock brings you the following features:
 
-### Run local
+- OpenAPI Mock uses [Swagger-Express-Middleware](https://github.com/BigstickCarpet/swagger-express-middleware) to parse, validate, and dereference OpenAPI files.  You can also create your custom implementation for responses or errors.
```

Comment:
Here we should also link to: https://github.com/APIDevTools/swagger-express-middleware/blob/master/docs/middleware/mock.md

Reply:
yes, something like: See xxx to learn how the API Mock will react to incoming requests

Reply:
Done.